ui/components/src/RefreshIntervalPicker/RefreshIntervalPicker.tsx

```diff
@@ -37,6 +38,12 @@ export function RefreshIntervalPicker(props: RefreshIntervalPickerProps) {
           inputProps={{
             'aria-label': `Select refresh interval. Currently set to ${formattedValue}`,
           }}
+          sx={{
+            maxWidth: 70,
+            '.MuiInputBase-inputSizeSmall': {
+              paddingLeft: 1,
+            },
+          }}
```

Comment:
The left padding looks OK to me right now (looking at my local perses with main brain) but you might be seeing something I'm not